heed-types/src/cow_slice.rs

```diff
@@ -58,10 +59,10 @@ where
                         vec.set_len(elems);
                     }
 
-                    return Some(Cow::Owned(vec));
+                    return Ok(Cow::Owned(vec));
                 }
 
-                None
+                Err("The provided bytes do not satisfy the alignment requirements.")?
```

Comment:
When I switch from zerocopy to bytemuck I will change this error string into [a nicer error, more precise](https://docs.rs/bytemuck/1.4.1/bytemuck/enum.PodCastError.html)!